build-bench-env.sh

```diff
@@ -663,6 +670,21 @@ if test "$setup_redis" = "1"; then
   popd
 fi
 
+if test "$setup_rocksdb" = "1"; then
+  phase "build rocksdb $version_rocksdb"
+
+  pushd $devdir
+  if test -d "rocksdb"; then
```

Comment:
You can replace this with the `checkout` function :)

Reply:
Done